src/app/records/PresentationNode.tsx

```diff
@@ -66,6 +69,10 @@ export default function PresentationNode({
   };
 
   const { visible, acquired, nodeDef } = node;
+  // Hack to indicate children are no longer obtainable
+  const isLegacy =
+    isParentLegacy ||
+    ['Legacy Triumphs', 'Legacy Seals'].includes(node.nodeDef.displayProperties?.name);
```

Comment:
Unfortunately this will only work with the English manifest - you'll need to use node definition hashes instead.

Reply:
You can find the hashes at https://data.destinysets.com/